package.json

```diff
@@ -4,6 +4,7 @@
   "version": "3.3.1",
   "homepage": "https://marionettejs.com/",
   "main": "lib/backbone.marionette.js",
+  "module": "lib/backbone.marionette.es.js",
```

Comment:
Is anything using esm yet?  There is also a lot of discussion on .mjs.  My leaning is: there currently seems to be no benefit to changing the file extension but it does potentially introduce complications with IDEs and syntax highlighting.  That will probably change in 8 months but for _today_ I don't see any benefit to a non '.js' extension.